code/hash/Hacl.Hash.SHA3.fsti

```diff
@@ -0,0 +1,21 @@
+module Hacl.Hash.SHA3
+
+// Just a few combinators needed to instantiate the streaming functor.
+
+open Spec.Hash.Definitions
+open Hacl.Hash.Definitions
+
+inline_for_extraction noextract
```

Comment:
just didn't want another file to appear in the code-gen so I marked everything as inline... another difference is that these functions just "redirect" to Hacl.SHA3, but with the agile hash signatures (so worth inlining since they just call absorb/squeeze/etc.); for Hacl.SHA2, the actual code is in there, so not a good idea to inline